Gemfile

```diff
@@ -3,16 +3,3 @@
 source 'http://rubygems.org'
 
 gemspec
-
-FOREMAN_GEMFILE = File.expand_path('.foreman_app/Gemfile', __dir__)
-if File.exist?(FOREMAN_GEMFILE)
-  instance_eval(Bundler.read_file(FOREMAN_GEMFILE))
-else
-  puts <<~MESSAGE
-    Foreman source code is not present. To get the latest version, run:
-
-      rake setup:foreman_prepare
-
-    and try again.
-  MESSAGE
-end
```

Comment:
This is no longer in our template, honestly I don't know what is this and if this ever worked. There is no Rakefile in this repo so the instructions are not correct.

https://github.com/theforeman/foreman_plugin_template/blob/master/Gemfile